spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,115 @@
+
+
+    ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+    ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id)
+    ContestantProject.create(contestant_id: gretchen.id, project_id: upholstery_tux.id)
+    ContestantProject.create(contestant_id: kentaro.id, project_id: upholstery_tux.id)
+    ContestantProject.create(contestant_id: kentaro.id, project_id: boardfit.id)
+    ContestantProject.create(contestant_id: erin.id, project_id: boardfit.id)
+    
+    visit "/projects/#{news_chic.id}"
+
+    expect(page).to have_content(news_chic.name)
+    expect(page).to have_content(news_chic.material)
+
+
+    visit "/projects/#{upholstery_tux.id}"
+
+    expect(page).to have_content(upholstery_tux.name)
+    expect(page).to have_content(upholstery_tux.material)
+    
+
+  end
+
+  it "displays the theme of the challenge this project belongs to" do
+    recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+
+    visit "/projects/#{news_chic.id}"
+
+    expect(page).to have_content(recycled_material_challenge.theme)
+  end
+
+  #user story 3
+  it "displays the number of contestants assigned to a project" do
+    recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+    news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    boardfit = recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+    upholstery_tux = furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    lit_fit = furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+    jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+
+    ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+    ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id)
+    ContestantProject.create(contestant_id: gretchen.id, project_id: upholstery_tux.id)
+    ContestantProject.create(contestant_id: kentaro.id, project_id: upholstery_tux.id)
+    ContestantProject.create(contestant_id: kentaro.id, project_id: boardfit.id)
+    ContestantProject.create(contestant_id: erin.id, project_id: boardfit.id)
+    
+    visit "/projects/#{news_chic.id}"
+
+    expect(page).to have_content("2")
```

Comment:
This test is fine, but the way Capybara works is, if you have a number 2 ANYWHERE on this page, this test will pass. If you hav ea timestamp on the page for example, and it's 12:30, this test will pass 🤷  Kinda weird, right? 

Not really, if you think about it -- we're telling it to check the `page` variable, which contains a stringified copy of all the content on the page. 
To fix this test, yo ucould either do 
```
expect(page).to have_content("Number of Contestants: 2")
```
that way it checks for that entire string that includes the number we're looking for. 
Or, you could use a `within` block: 
```
within "#num_contestants" do
   expect(page).to have_content(2)
end
```
But this would only work if there's an HTML ID on the page of `num_contestants`. Try these out to see what I mean!